stacker/tests/test_environment.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from stacker.environment import parse_environment
+
+test_env = """key1: value1
```

Comment:
Per above - add blank lines & comments just to make sure we don't miss anything.